### PR TITLE
[ACCEL] Add email notifications for ora2

### PR DESCRIPTION
**Pull request by arsentur**
**Description:**
Add email notifications when staff:
- canceled assessment
- returned assessment
- Mark as done assessment

Add translations for email messages

**YouTrack:**
https://youtrack.raccoongang.com/issue/ACCEL-233

### CODE REVIEW

File: openassessment/utils/email_notification.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    try:
+        send_mail(
+            subject=email_action_dict.get(action_name).get("subject"),
+            message=render_to_string(email_action_dict.get(action_name).get("template"), email_params),
+            from_email=from_email,
+            recipient_list=[user_email],
+            fail_silently=False
+        )
+    except SMTPException:
+        log.warning("Failure sending ORA staff response form e-mail")
```
Comment (flying-pi) [inline openassessment/utils/email_notification.py]: I don`t sure about is this type of message. As I remember it doesn`t send to the production log. Could you switch to the error message?

File: openassessment/xblock/staff_area_mixin.py [hunk 2]
```diff
@@ -250,6 +251,26 @@ def render_staff_grade_counts(self, data, suffix=''):  # pylint: disable=W0613
         except PeerAssessmentInternalError:
             return self.render_error(self._(u"Error getting staff grade ungraded and checked out counts."))
 
+    def get_user_email_by_submission_uuid(self, submission_uuid):
```
Comment (flying-pi) [inline openassessment/xblock/staff_area_mixin.py]: Not enough clear for me. Could you slit this function it to 2 separate functions?

File: openassessment/xblock/staff_assessment_mixin.py [hunk 3]
```diff
@@ -27,6 +28,26 @@ def staff_assessment_exists(self, submission_uuid):
 
         return staff_api.get_latest_staff_assessment(submission_uuid) is not None
 
+    def get_user_email_by_submission_uuid(self, submission_uuid):
```
Comment (flying-pi) [inline openassessment/xblock/staff_assessment_mixin.py]: Seems it copy paste error.

File: openassessment/xblock/staff_area_mixin.py (hunk 2 above)
Comment (arsentur) [reply openassessment/xblock/staff_area_mixin.py]: Done

File: openassessment/xblock/staff_assessment_mixin.py (hunk 3 above)
Comment (arsentur) [reply openassessment/xblock/staff_assessment_mixin.py]: I created new Mixin with helper method and removed these duplicate methods.

File: openassessment/utils/email_notification.py (hunk 1 above)
Comment (arsentur) [reply openassessment/utils/email_notification.py]: Done

**Pull request merged**